README.md

```diff
@@ -1326,7 +1326,36 @@ Work in progress crates. Help the authors make these crates awesome!
+1. [`no-panic`]: Attribute macro to require that the compiler prove a function can't ever panic. [![crates.io](https://img.shields.io/crates/v/no-panic.svg)](https://crates.io/crates/no-panic)
+1. [`panic-abort`]: Causes an abort on panics (nightly-only) [![crates.io](https://img.shields.io/crates/v/panic-abort.svg)](https://crates.io/crates/panic-abort)
+1. [`panic-halt`]: Halts the processor on panics [![crates.io](https://img.shields.io/crates/v/panic-halt.svg)](https://crates.io/crates/panic-halt)
+1. [`panic-itm`]: Prints the panic through the chip's ITM to a debugger attached to the SWO pin (Cortex-M with ITM only)
+1. [`panic-never`]: This crate guarantees that your application is free of panicking branches. [![crates.io](https://img.shields.io/crates/v/panic-never.svg)](https://crates.io/crates/panic-never)
+1. [`panic-persist`]: A panic handler crate inspired by `panic-ramdump` that logs panic messages to a region of RAM defined by the user, allowing for discovery of panic messages post-mortem using normal program control flow. - [![crates.io](https://img.shields.io/crates/v/panic-persist.svg)](https://crates.io/crates/panic-persist)
+1. [`panic-probe`]: A panic handler that exits `probe-run` with an error code. [![crates.io](https://img.shields.io/crates/v/panic-probe.svg)](https://crates.io/crates/panic-probe)
+1. [`panic-ramdump`]: Writes the panic message into the beginning of RAM and enters an infinite loop; the message can be extracted by attaching a debugger (Cortex-M only) [![crates.io](https://img.shields.io/crates/v/panic-ramdump.svg)](https://crates.io/crates/panic-ramdump)
+1. [`panic-rtt`]: Set the panicking behavior to log to a JLINK debugger and loop. [![crates.io](https://img.shields.io/crates/v/panic_rtt.svg)](https://crates.io/crates/panic_rtt)
+1. [`panic-rtt-target`]: Logs panic messages over RTT. Uses `rtt-target`. [![crates.io](https://img.shields.io/crates/v/panic-rtt-target.svg)](https://crates.io/crates/panic-rtt-target)
+1. [`panic-semihosting`]: Prints the panic message via semihosting to an attached debugger (Cortex-M only)
+
+[`dont_panic_slice`]: https://crates.io/crates/dont_panic_slice
+[`dont_panic`]: https://crates.io/crates/dont_panic
+[`no-panic`]: https://crates.io/crates/no-panic
+[`panic-abort`]: https://crates.io/crates/panic-abort
+[`panic-halt`]: https://crates.io/crates/panic-halt
+[`panic-itm`]: https://github.com/rust-embedded/cortex-m/tree/master/panic-itm
+[`panic-never`]: https://crates.io/crates/panic-never
+[`panic-persist`]: https://crates.io/crates/panic-persist
+[`panic-probe`]: https://crates.io/crates/panic-probe
+[`panic-ramdump`]: https://crates.io/crates/panic-ramdump
+[`panic-rtt`]: https://crates.io/crates/panic_rtt
+[`panic-rtt-target`]: https://crates.io/crates/panic-rtt-target
+[`panic-semihosting`]: https://github.com/rust-embedded/panic-semihosting
```

Comment:
Link to crates.io here instead? https://crates.io/crates/panic-semihosting

Reply:
I made links to crates io for crates that are regularly updated, these are kept as links to github because their crates.io versions are significantly behind.